Tensile/SolutionStructs.py

```diff
@@ -1537,6 +1537,16 @@ def assignDerivedParameters(state):
           reject(state, "GlobalLoadVectorWidthB %u must be == VectorWidth %u or == 1" % \
                   (state["GlobalLoadVectorWidthB"], state["VectorWidth"]))
 
```

Comment:
I am not sure if I understand  the logic,  I was thinking it would be something like:
if state["KernelLanguage"] == "Assembly" and state["VectorWidth"] > 1:
  state["AssertMinApproxSize"] = 2
else:
  state["AssertMinApproxSize"] = 0

Reply:
Can you review the final version:

    if state["KernelLanguage"] == "Assembly":
      # Asm kernels only work if all dims are > 32
      state["AssertMinApproxSize"] = 1
    elif state["VectorWidth"] > 1:
      # VW>1 kernels require dims>1
      state["AssertMinApproxSize"] = 2
    else:
      # these work everywhere, no special restrictions
      state["AssertMinApproxSize"] = 0